bucket/ludusavi.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "0.3.0",
+    "description": "PC video game save data backup tool",
+    "homepage": "https://github.com/mtkennerly/ludusavi",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/mtkennerly/ludusavi/blob/master/LICENSE"
+    },
```

Comment:
@Ash258 had requested this change in [another comment](https://github.com/lukesampson/scoop-extras/pull/4413#discussion_r454143156), but it got marked as resolved since it was also part of the vcredist discussion. I may be mistaken, but I think this part of the comment probably still applies:

```
    "license": "MIT",
```

Reply:
Not needed.. I stil need to finish this PR by myself so I will do it.